config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+	devise_for :users, :controllers => {registrations: 'registrations'}
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources 'posts'
```

Comment:
As asked [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_routes_requirement.png), the application routes are `resources :posts, only: [:new, :create, :index]`